validation/mounts.go

```diff
@@ -1,9 +1,16 @@
 package main
 
 import (
+	"fmt"
+
 	"github.com/opencontainers/runtime-tools/validation/util"
 )
 
 func main() {
-	util.Skip("TODO: mounts generation options have not been implemented", "")
```

Comment:
This test is for runtime validation (and the TODO is stale now that #279 has landed, but that's another matter).

Tests for *generation* functionality (which is what you're adding in this PR) should go in [`generate/generate_test.go`][1].

[1]: https://github.com/opencontainers/runtime-tools/blob/c3755c1175a1baad7230dfb03751e3a4b7182f32/generate/generate_test.go

Reply:
I am not sure why you would not want to have this in the validation code?  This seems like the correct place unless you are dropping the validation code also?

Reply:
> I am not sure why you would not want to have this in the validation code?

There are several types of validation in this repo:

1. Config validation in `validate`, with unit tests in `validate/validate_test.go`.
2. Runtime validation (which calls the runtime executable, e.g. [here][1]) in `validation`.  There are no unit tests for this (yet?).
3. Config generation argument checks in `generate`, with unit tests (of which there are currently few) in `generate/generate_test.go`.

The tests you're currently adding here (a type-2 location) belong to type-3 (generation unit tests).

[1]: https://github.com/opencontainers/runtime-tools/blob/c3755c1175a1baad7230dfb03751e3a4b7182f32/validation/create.go

Reply:
Ok copied it to the generate_test.go.  But not sure if I should remove it totally from Validate.  Currently tests in both places.

Reply:
> Ok copied it to the generate_test.go.

That looks good to me.

> But not sure if I should remove it totally from Validate.

I think you should, because the test has nothing to do with runtime compliance (which is what `validation/` is for).  It's just testing the generate implementation, so the `generate_test.go` test you've added is all you need.